### PR TITLE
Added documentation for compute_shuffled_index.

### PR DESCRIPTION
**Pull request by HristoStaykov**

### CODE REVIEW

**Review by Owliie — CHANGES_REQUESTED** [review]

File: vendor/zkllvm-metacraft-circuits/src/circuits/CMakeLists.txt [hunk 1]
```diff
@@ -3,3 +3,9 @@ add_example(weigh_justification_and_finalization SOURCES weigh_justification_and
 
 add_dependencies(template compute_shuffled_index)
 add_dependencies(template weigh_justification_and_finalization)
+
+add_custom_command(TARGET compute_shuffled_index POST_BUILD
+    COMMAND cp -r ${CMAKE_CURRENT_SOURCE_DIR}/compute_shuffled_index.json ${CMAKE_CURRENT_BINARY_DIR}
```
Comment (Xearty) [inline vendor/zkllvm-metacraft-circuits/src/circuits/CMakeLists.txt:8]: You can delete the recursive flag

File: vendor/zkllvm-metacraft-circuits/src/circuits/CMakeLists.txt [hunk 2]
```diff
@@ -3,3 +3,9 @@ add_example(weigh_justification_and_finalization SOURCES weigh_justification_and
...
+    DEPENDS compile_commands.json
```
Comment (Xearty) [inline vendor/zkllvm-metacraft-circuits/src/circuits/CMakeLists.txt:9]: Check if this dependency is required. POST_BUILD is probably enough

Comment (HristoStaykov) [reply vendor/zkllvm-metacraft-circuits/src/circuits/CMakeLists.txt:9]: It is strange, but it is required by cmake.

**Review by Owliie — APPROVED** [review]

**Review by Xearty — DISMISSED** [review]

**Comment by Xearty** [comment]
This was merged 2 weeks ago in https://github.com/metacraft-labs/DendrETH/pull/291

**Pull request closed without merging**